com.github.chipmuenk.pyfda.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: com.github.chipmuenk.pyFDA
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: pyfdax
+finish-args:
+- "--socket=wayland"
+- "--socket=x11"
+- "--device=dri"
+- "--filesystem=xdg-run/dconf"
+- "--filesystem=~/.config/dconf:ro"
+- "--talk-name=ca.desrt.dconf"
+- "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+- "--own-name=com.github.chipmuenk.pyFDA"
+- "--filesystem=home"
+rename-icon: pyfda_icon #Image will renamed to match the app-id konvention
+rename-appdata-file: pyFDA.appdata.xml
+rename-desktop-file: pyFDA.desktop
+modules:
+- name: PyQt5
+  cleanup:
+  - "/bin/sip"
+  - "/include"
+  - "/lib/python3.7/site-packages/*.pyi"
```

Comment:
the latest runtime use python 3.9 so this will have to change

Reply:
done

Reply:
not relevant anymore